src/main/java/com/deviceinsight/kafka/health/config/KafkaHealthProperties.java

```diff
@@ -43,7 +43,7 @@ public void setSubscriptionTimeout(Duration subscriptionTimeout) {
 
 	@Override
 	public String toString() {
-		return "KafkaHealthProperties{" + "topic='" + topic + '\'' + ", sendReceiveTimeout=" + sendReceiveTimeout +
-				", pollTimeout=" + pollTimeout + ", subscriptionTimeout=" + subscriptionTimeout + '}';
+		return "KafkaHealthProperties{" + "topic='" + topic + '\'' + ", sendReceiveTimeout=" + sendReceiveTimeout
+				+ ", pollTimeout=" + pollTimeout + ", subscriptionTimeout=" + subscriptionTimeout + '}';
 	}
 }
```

Comment:
Please revert the changes to this file as they're unnecessary.

Reply:
I moved this file to separate package. Health.config
Should this be reverted.